<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PoioILE</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMain" stretch="0,0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutTierWords">
          <item>
           <widget class="QLabel" name="labelWords">
            <property name="text">
             <string>Word tier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetWords">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushbuttonNewTierWords">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutUtterances">
          <item>
           <widget class="QLabel" name="labelUtterances">
            <property name="text">
             <string>Utterances tier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetUtterances">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushbuttonNewTierUtterances">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutMorphemes">
          <item>
           <widget class="QLabel" name="labelMorphemes">
            <property name="text">
             <string>Morpheme tier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetMorphemes">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushbuttonNewTierMorphemes">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutFunctions">
          <item>
           <widget class="QLabel" name="labelFunctions">
            <property name="text">
             <string>Gloss tier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetFunctions">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushbuttonNewTierFunctions">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutTranslations">
          <item>
           <widget class="QLabel" name="labelTranslations">
            <property name="text">
             <string>Translation tier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetTranslations">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushbuttonNewTierTranslations">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutEditArea">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutProjectFiles">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Project Files:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listwidgetFiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="toolbuttonAddFile">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Add File...</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="poio.qrc">
                <normaloff>:/images/pixmaps/fileopen.png</normaloff>:/images/pixmaps/fileopen.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolbuttonNewFile">
              <property name="toolTip">
               <string>New File</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="poio.qrc">
                <normaloff>:/images/pixmaps/filenew.png</normaloff>:/images/pixmaps/filenew.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolbuttonExportFile">
              <property name="toolTip">
               <string>Export File...</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="poio.qrc">
                <normaloff>:/images/pixmaps/fileexport.png</normaloff>:/images/pixmaps/fileexport.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolbuttonRemoveFile">
              <property name="toolTip">
               <string>Remove File</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="poio.qrc">
                <normaloff>:/images/pixmaps/fileclose.png</normaloff>:/images/pixmaps/fileclose.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PoioIlTextEdit" name="texteditInterlinear"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionAddFile"/>
    <addaction name="actionNewFile"/>
    <addaction name="actionExportFile"/>
    <addaction name="actionRemoveFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopyUtterance"/>
    <addaction name="separator"/>
    <addaction name="actionInsertUtterance"/>
    <addaction name="actionDeleteUtterance"/>
    <addaction name="actionInsertWord"/>
    <addaction name="actionDeleteWord"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutPoioILE"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionInsertUtterance"/>
   <addaction name="actionDeleteUtterance"/>
   <addaction name="actionInsertWord"/>
   <addaction name="actionDeleteWord"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/projectopen.png</normaloff>:/images/pixmaps/projectopen.png</iconset>
   </property>
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/filesave.png</normaloff>:/images/pixmaps/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/filesaveas.png</normaloff>:/images/pixmaps/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save File as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/projectnew.png</normaloff>:/images/pixmaps/projectnew.png</iconset>
   </property>
   <property name="text">
    <string>New Project...</string>
   </property>
   <property name="toolTip">
    <string>Create a new annotation project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDeleteUtterance">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/removeutterance.png</normaloff>:/images/pixmaps/removeutterance.png</iconset>
   </property>
   <property name="text">
    <string>Delete utterance</string>
   </property>
  </action>
  <action name="actionInsertUtterance">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/insertutterance.png</normaloff>:/images/pixmaps/insertutterance.png</iconset>
   </property>
   <property name="text">
    <string>Insert new utterance</string>
   </property>
   <property name="toolTip">
    <string>Insert new utterance</string>
   </property>
  </action>
  <action name="actionCopyUtterance">
   <property name="text">
    <string>Copy utterance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionInsertWord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/insertword.png</normaloff>:/images/pixmaps/insertword.png</iconset>
   </property>
   <property name="text">
    <string>Insert new word</string>
   </property>
  </action>
  <action name="actionDeleteWord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/removeword.png</normaloff>:/images/pixmaps/removeword.png</iconset>
   </property>
   <property name="text">
    <string>Delete word</string>
   </property>
  </action>
  <action name="actionAboutPoioILE">
   <property name="text">
    <string>About PoioILE...</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/fileopen.png</normaloff>:/images/pixmaps/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Add File...</string>
   </property>
  </action>
  <action name="actionExportFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/fileexport.png</normaloff>:/images/pixmaps/fileexport.png</iconset>
   </property>
   <property name="text">
    <string>Export File...</string>
   </property>
  </action>
  <action name="actionRemoveFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/fileclose.png</normaloff>:/images/pixmaps/fileclose.png</iconset>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="poio.qrc">
     <normaloff>:/images/pixmaps/filenew.png</normaloff>:/images/pixmaps/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New File..</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PoioIlTextEdit</class>
   <extends>QTextEdit</extends>
   <header>PoioIlTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="poio.qrc"/>
 </resources>
 <connections/>
</ui>
